analyzer/src/funTest/kotlin/DotNetSupportTest.kt

```diff
@@ -18,7 +18,7 @@
  * License-Filename: LICENSE
  */
 
-package com.here.ort.analyzer
+package com.here.ort.analyzer.managers.utils
```

Comment:
The locations of the test classes should correspond to the locations of the classes under test, so this file should also be moved to a "managers/utils" directory. Same applies to the package manager test classes which are currently not in a "managers" directory. Maybe want to fix this in a separate PR?

Reply:
No, I can do it as part of this PR. Would you then nest "integration" also under "managers"?

Reply:
I've decided to keep "integration" at the same level as "managers".